Dockerfile

```diff
@@ -29,6 +29,12 @@ RUN set -ex; \
   yarn build; \
   chmod +x dist/*.js;
 
+# hardcode node version to renovate
+RUN set -ex; \
+  NODE_VERSION=$(node -v | cut -c2-); \
```

Comment:
Is NODE_VERSION likely to get used or confused anywhere else? Eg name it DEFAULT_NODE_VERSION instead?